CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changes
 
+## 0.6.5
```

Comment:
Let's make this `UNRELEASED`. We only replace the actual number when publishing to npm.